app/react/components/form-components/Slider/Slider.tsx

```diff
@@ -23,29 +26,32 @@ export function Slider({
   dataCy,
   visibleTooltip: visible,
 }: Props) {
```

Comment:
Is it possible to mention to testers all the places that this component is used for regression testing?

Reply:
Yep good idea!